lib/facter/rh_certificates.rb

```diff
@@ -0,0 +1,26 @@
+CERT_STRING = '/cert.pem'
```

Comment:
Constant are globals in this case, maybe make sure you create them with a unique namespace.